src/config/theme.ts

```diff
@@ -245,6 +245,21 @@ const createThemeOptions = (theme: Theme) => ({
         },
       },
     },
+    MuiFormLabel: {
+      styleOverrides: {
+        root: theme.unstable_sx({
+          typography: 'body2',
+          fontWeight: 600,
```

Comment:
Fields rendered within a DynamicForm get special labels ([RequiredLabel] component(https://github.com/greenriver/hmis-frontend/blob/201900cd61d2e961a957c4a494c8110f950bcd03/src/modules/form/components/RequiredLabel.tsx#L17)). Here I update the theme to match that style (body2 bold), so that inputs rendered _outside_ of dynamic form match the same styles as dynamic forms.